Facepunch.Steamworks/Interfaces/Workshop.cs

```diff
@@ -123,6 +123,37 @@ public Item GetItem( ulong itemid )
             return new Item( itemid, this );
         }
 
+        /// <summary>
+        /// The amount of item's the user is subscribed to for this App
+        /// </summary>
+        public uint GetAmountSubsribed()
+        {
+            return ugc.GetNumSubscribedItems();
+        }
+        /// <summary>
+        /// Creates a list of all item's that the user is currently subscribed
+        /// to for this App. This does not query titles or descriptions but
+        /// allows you to get the directories, state and ID of any subscribed Item.
+        /// This is mostly useful for getting all subscribed items install location.
+        /// </summary>
+        public Item[] GetSubscribedItems(uint amount)
```

Comment:
I kind of doubt the usefulness of this function, plus why would you ever use the amount parameter?